Projects/Contracts/Sources/Stores/ContractAction.swift

```diff
@@ -34,11 +34,13 @@ public enum ContractLoadingAction: LoadingProtocol {
 public enum EditType: String, Codable, Hashable, CaseIterable {
     case changeAddress
     case coInsured
+    case changeTier
 
     var title: String {
         switch self {
         case .coInsured: return L10n.contractEditCoinsured
         case .changeAddress: return L10n.InsuranceDetails.changeAddressButton
+        case .changeTier: return "Change Tier"
```

Comment:
I know this is just for staging but would nice to have this one localized anyway :)

Reply:
@sladan-hedvig maybe just create the ket and come up with something for now and then Kristofer can change it later